mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -140,7 +140,11 @@ class CrossValidator(override val uid: String) extends Estimator[CrossValidatorM
 @Experimental
 class CrossValidatorModel private[ml] (
     override val uid: String,
+
+    /** The best model selected from cross validation */
```

Comment:
Those are params for the class constructor.  I'd recommend you try generating the Scala docs as you have the code now, and with the docs moved to param docs, to see the difference.